### PR TITLE
Document who pays the transaction fee

### PR DESCRIPTION
**Pull request by ThomasArts**

### CODE REVIEW

File: channels/ON-CHAIN.md [hunk 1]
```diff
@@ -104,34 +104,35 @@ Serialization defined [here](../serializations.md#channel-deposit-transaction)
...
+account should hold enough tokens to pay the fee, i.e., the fee is subtracted before
+the withdrawn tokens arrive.
 
 Serialization defined [here](../serializations.md#channel-withdraw-transaction)
 
 
 - `channel_id`: channel id as recorded on chain
-- `from`:
-- `amount`:
+- `from`: receiver of the withdraw
```
[the remark this answers is not in the document]
Comment (lucafavatella) [reply channels/ON-CHAIN.md]: (also in paragraph above)

Code prefers `to` too https://github.com/aeternity/epoch/blob/325a1bfe25a2c69be39985059ed4c3cc7e854c65/apps/aechannel/src/aesc_withdraw_tx.erl#L184

Comment (lucafavatella) [inline channels/ON-CHAIN.md]: I see - "receiver of the withdraw" means "sender of the transaction, receiver of the withdrawn tokens".

Comment (ThomasArts) [reply channels/ON-CHAIN.md]: Well spotted, I bumped into it while testign as well. Update on its way.

File: channels/ON-CHAIN.md [hunk 2]
```diff
@@ -88,7 +88,7 @@ channel_id = Blake2b(initiator || channel_create_tx_nonce || responder)
 ### `channel_deposit`
 
 Depositing funds into a channel after creation should allow channels to be more
-long lived due to the increased ease of balancing them out. The amount of coins
+long lived due to the increased ease of balancing them out. The amount of tokens
```
[the remark this answers is not in the document]
Comment (ThomasArts) [reply channels/ON-CHAIN.md]: I remember a long discussion in Matla where we decided to call it aeons or so, definitely not coins.

[the remark this answers is not in the document]
Comment (knarz) [reply channels/ON-CHAIN.md]: Aeons are the coins for aeternity. Coin is not the name, just the description.

[the remark this answers is not in the document]
Comment (ThomasArts) [reply channels/ON-CHAIN.md]: So, shall I replace "tokens" in the document by "aeons" or rather "coins"?

[the remark this answers is not in the document]
Comment (knarz) [reply channels/ON-CHAIN.md]: My personal preference would be coin whenever we're talking about abstract usage. In the case of specific amounts, I'd use aeons.

Does that make sense?

**Pull request merged**